resources/todomvc/architecture-examples/react-redux/src/components/text-input.jsx

```diff
@@ -0,0 +1,55 @@
+import { Component } from "react";
+import classnames from "classnames";
+import PropTypes from "prop-types";
+
+export default class TextInput extends Component {
+    static propTypes = {
+        onSave: PropTypes.func.isRequired,
+        text: PropTypes.string,
+        placeholder: PropTypes.string,
+        editing: PropTypes.bool,
+        newTodo: PropTypes.bool,
```

Comment:
It's not obvious what  `newTodo` does, I think a comment would be useful for this property.

Reply:
added a note 😄